scene/2d/camera_2d.h

```diff
@@ -147,6 +147,7 @@ class Camera2D : public Node2D {
 	Vector2 get_zoom() const;
 
 	Point2 get_camera_screen_center() const;
+	Size2 get_camera_screen_size() const;
```

Comment:
I think it does not need to be a public function. You should move it to the private section and prepend it with an underscore.

Reply:
fixed!